src/Components/test/testassets/MegaTestApp/MegaTestApp.csproj

```
@@ -0,0 +1,61 @@
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+
+    <!-- Must be defined before ReferenceFromSource.props is imported -->
+    <AdditionalRunArguments>--pathbase /subdir</AdditionalRunArguments>
+
+    <!-- Resx generation on Resources.resx only -->
+    <GenerateResxSource>false</GenerateResxSource>
+
+    <!-- Project supports more than one language -->
+    <BlazorWebAssemblyLoadAllGlobalizationData>true</BlazorWebAssemblyLoadAllGlobalizationData>
+    <_UseRazorSourceGenerator>false</_UseRazorSourceGenerator>
```

```suggestion
    <UseRazorSourceGenerator>false</UseRazorSourceGenerator>
```

This outta do the trick. We updated the property name in a recent SDK version.